source/manual/replicate-app-data-locally.html.md

```diff
@@ -18,20 +18,86 @@ repository](https://github.com/alphagov/govuk-puppet).
 
-* access to integration; or
+* access to integration via aws; or
 * database exports from someone that does.
 
+## AWS access
+
+The [aws setup guide](/manual/user-management-in-aws.html) covers using the interface in full, however if you only
+want to do developer replication you will need to:
+
+1. Log into AWS
+
+You should have received a email requesting you do this. If you have multiple AWS accounts ensure you are logged into `gds-users`
+
+1. Setup MFA/2FA on your devise
+
+Naviagte to `IAM -> Users -> <Your username>`
+
+If you don't see an ARN ID next to `Assigned MFA device` click the edit button and set one up.
+
+1. Create an access token
+
+Under `Access keys`click `Create access key` - you will secret is only displayed once, however if you fail to note it down just remove it and create another.
+
+1. Install the AWS CLI application locally
+
+On MacOS this can be done using `brew install awscli`. This is being installed locally with the assumption you will be doing a two step backup process, i.e. download the files on your local machine and then update you VM. This is the recommended process as the download is quicker to you local machine for most users.
+
+1. Setup your AWS access config and credentials files
+
+I am repeating the instructions desribed in the [aws setup guide](/manual/user-management-in-aws.html) with one difference. In the credentials file the setting must be under default as otherwise they are not found by the replication script.
+
+Create a `~/.aws/config` file:
+
+```
+[profile govuk-integration]
+role_arn = <Role ARN>
+mfa_serial = <MFA ARN>
+source_profile = gds
+region = eu-west-1
+```
+
+Role ARN: should be one of the following depending on the level of access you have been configured with [here](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/integration/common.tfvars):
+
+* govuk-admins: arn:aws:iam::210287912431:role/govuk-administrators
+* govuk-poweruser: arn:aws:iam::210287912431:role/govuk-poweruser
+* govuk-users: arn:aws:iam::210287912431:role/govuk-users
+
+**MFA ARN**: This is the long string next to `Assigned MFA device` in the AWS console
+
+Create a `~/.aws/credentials` file
+
+```
+[default]
+aws_access_key_id = <access key id>
+aws_secret_access_key = <secret access key>
+```
+
+> Please note this file differs in the it uses `[default]` instead of `[gsd]`.
+
+**access key id**: This is the ID that we created early
+**secret access key**: This is the secret associated with the key we created earlier. If you didn't note these down you can simply create a new one.
+
 ## If you have integration access
 
 If you have integration access, you can download and import the latest data by
 running:
 
-    dev$ cd /var/govuk/govuk-puppet/development-vm/replication dev$
-    ./replicate-data-local.sh -u $USERNAME -F ../ssh_config
+    dev$ cd /var/govuk/govuk-puppet/development-vm/replication
+    dev$ ./replicate-data-local.sh -u $USERNAME -F ../ssh_config -a <MFA code>
```

Comment:
If we're running the download on the mac as we advise further up, then this could be:

```
mac$ cd ~/govuk/govuk-puppet/development-vm/replication
mac$ ./replicate-data-local.sh -u $USERNAME -n -F ../ssh_config -a <MFA code>
```
then 
```
dev$ cd /var/govuk/govuk-puppet/development-vm/replication
dev$ ./replicate-data-local.sh -d path/to/dir -s
```